experimental/PropertyDDS/packages/property-shared-tree-interop/package.json

```diff
@@ -72,7 +76,8 @@
 		"tasks": {
 			"eslint": [
 				"build:esnext",
-				"build:test"
+				"build:test",
+				"build:test:esm"
```

Comment:
This one (build:test:esm) shouldn't be needed since build:test already depends on it. Did your testing behave otherwise?

Reply:
policy-check required it, but I suspect that's because of the other pre-existing task dependencies.

It's not obvious to me why this package needs special task dependencies.  I'll try removing them.